constants.h

```diff
@@ -4,7 +4,7 @@
 #include <cmath>
 
 namespace constants {
-const float kAccuracy = 0.000005;
+const float kAccuracy = 0.00005;
```

Comment:
Why?

Reply:
Sounds good. Please add a comment that this accuracy is expressed in meters.

Right now it is a generic "accuracy" for god knows what. With this you should go through all of the places where you use it and check if it make sense to use it, given you have semantics now.

Reply:
done. I also deleted another HitAccuracyConstants, becuase its not needed. One constant on accuray is enough ;)

Reply:
Comment needs some fixing still. Grammar does not seem to be correct still, and there is definition by repetition "accuracy is accuracy", and also it may not be true with the hit position bit (I do not see it there btw).

edit: I wanted to suggest a better comment, but could not as it is currently not clear what this is used for.

Please at least fix invalid English grammar and make sure this sentence is ok in Polish as well, as now it does not seem right.